docker/Dockerfile.testing

```diff
@@ -12,13 +11,18 @@ USER root
 # Keep same channel ordering to not revert anything.
 RUN umask 0000 && \
     conda config --remove channels defaults || true && \
-    pip uninstall -y ravenpy birdhouse-birdy && \
-    pip install --no-cache-dir --no-deps ravenpy==0.13.1 birdhouse-birdy==0.8.7 && \
-    mamba install -c conda-forge -c cdat -c bokeh -c fortiers -n birdy jupyterlab-git==0.44.0 && \
+    pip uninstall -y hsclient && \
+    mamba install --force-reinstall -c conda-forge -c cdat -c bokeh -c fortiers ravenpy==0.18.1 sparse==0.15.5 && \
```

Comment:
Looks like `sparse==0.15.5` did not take effect because in commit "py311-250415-update250416 full build artifacts" (https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/blob/b3d2c445512c82587118aebe5b938d4a09b72a19/docker/saved_buildout/conda-env-export.yml) we have `sparse=0.16.0=pyh1646c32_0` !

When you use `Dockerfile.testing`, you have to build the docker image locally (`docker build -t testing -f Dockerfile.testing .`) to test that you are pulling exactly the versions you want before triggering the official build on DockerHub.

Probably Jenkins still fail because of the wrong `sparse` version.